### PR TITLE
fix: remove blocked code and correct the func

### PR DESCRIPTION
**Pull request by sonle-geekyants**
**- What I did**
- Fixed search method in desktop version

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: lib/desktop_screens_new/history_screen/history_desktop.dart [hunk 1]
```diff
@@ -64,8 +64,6 @@ class _HistoryDesktopScreenState extends State<HistoryDesktopScreen> {
               filteredFiles = getDisplayFileData(provider.allFilesHistory);
             },
             successBuilder: (provider) {
-              filteredFiles = getDisplayFileData(provider.allFilesHistory);
```
Comment (sachins-geekyants) [inline lib/desktop_screens_new/history_screen/history_desktop.dart:67]: Why is this removed ?
It will not refresh history screen on receiving new file.

Comment (sonle-geekyants) [reply lib/desktop_screens_new/history_screen/history_desktop.dart:67]: @sachins-geekyants If you put this code here, it will block searching method
